books/RayTracingInOneWeekend.html

```diff
@@ -319,7 +319,7 @@
 Now we can change our main to use this:
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
-    #include "common/vec3.h"
+    #include "vec3.h"
```

Comment:
overall happy with this change, though i fully support a common folder structure, which could be illustrated by a more experienced programmer than myself.